apps/omg_api/lib/application.ex

```diff
@@ -53,7 +53,7 @@ defmodule OMG.API.Application do
              %{
                synced_height_update_key: :last_exiter_eth_height,
                service_name: :exiter,
-               block_finality_margin: block_finality_margin,
+               block_finality_margin: 0,
```

Comment:
Why 0? There will be no block_finality margin?

Reply:
yes, for the child chain, for exits we want to remove them from UTXO set immediately, regardless of reorgs etc. I'll make an explicit comment.

Not sure how this margin managed to slip through in fact.